Mathlib/Algebra/Order/Ring/Basic.lean

```diff
@@ -121,6 +121,14 @@ lemma Bound.pow_le_pow_right_of_le_one_or_one_le (h : 1 ≤ a ∧ n ≤ m ∨ 0
 
 end OrderedSemiring
 
+-- See note [reducible non instances]
+/-- Turn an ordered domain into a strict ordered ring. -/
+abbrev OrderedRing.toStrictOrderedRing (α : Type*) [OrderedRing α] [IsDomain α] :
```

Comment:
You only need `NoZeroDivisors α` here. Also, should it be a low priority instance?

Reply:
Done the first part. @YaelDillies recommended to me to not make this an instance, due to potential for looping (if I understand right).